.changeset/real-onions-move.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-app-invoices": patch
+---
+
+Update app-sdk to 0.38.0
```

Comment:
Missing notion of the addition of metadata migration scripts

Reply:
Migration fact is not changelog, actually its the opposite - similar to refactor its purpose is to be *not* noticeable, here is some short cheatsheet on changelogs

https://keepachangelog.com/en/1.0.0/